### PR TITLE
index commit

### PR DESCRIPTION
**Pull request by anonymoustaco**
final (hopefully index commit)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +0,0 @@
-# collegestationcodesociety.github.io
```
Comment (OrdoFlammae) [inline README.md:1]: Could you replace this file?

**Review by OrdoFlammae — CHANGES_REQUESTED** [review]
Looks very nice! Replace `favicon.gif` with `favicon.png` (@KateCat-attempts should be able to provide a conversion), replace the README, and we should be good to go!

**Comment by anonymoustaco** [comment]
here is the commit with favicon in PNG format and READNE added

**Comment by OrdoFlammae** [comment]
Looks great! Good job @anonymoustaco!

**Pull request merged**